src/api/objects/MutableDomainObject.js

```diff
@@ -0,0 +1,126 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2019, United States Government
```

Comment:
Daaaaamn. You started working on this in 2019? Mad props

Reply:
Yeah, good catch 😄  It's long overdue. Started as a branch of topic-core-refactor, kept getting out of date, and would take a couple of hours each time to get everything working again.

The issue it fixes is actually even older than this and dates back to [an epic debate in a PR from 2016](https://github.com/nasa/openmct/pull/1131#issuecomment-240855549)...